lua-plugins/cli_filter.lua.in

```diff
@@ -17,6 +17,29 @@
 local git_date = '$Date$'
 local git_revision = '$Revision$'
 
+-- Access constraints mapping to partition
+local constraints_mapping = {
+    askaprt = {
+        askaprt = 'askaprt'
+    },
+    copy = {
+        copy    = 'copy'
+    },
+    cpu = {
+        debug   = 'debug',
+        highmem = 'highmem',
+        work    = 'work'
+    },
+    gpu = {
+        debug   = 'gpu-dev',
+        highmem = 'gpu-highmem',
+        work    = 'gpu'
+    },
+    ['mwa-asvocopy'] = {
```

Comment:
Is there a reason to have `mwa-asvocpy` be different from the rest?

Reply:
yeah, it's called LUA :(